### ISSUE TITLE
Class' instance variables (attributes) naming

### ISSUE THREAD

**Issue by Jtachan** [astral-sh/ruff#20435]
### Summary

I can't find any rule about naming the attributes within a class. While there is no specific PEP rule about this within the index, this is documented at the PEP8 style guide: https://peps.python.org/pep-0008/#method-names-and-instance-variables

I've double checked, and right now the `N` rules do warn in the following scenario:

```python
class MyClass:
    def __init__(self):
        self.invalidPepNaming = ...
```

**Comment by ntBre** [comment]
Thanks for the report! I believe this is a duplicate of #19893. Based on my comment there, I think this is consistent with the upstream pep8-naming linter, but we could still consider expanding the checks at some point.
